frontend/packages/topology/integration-tests/features/topology/export-application/export-of-applicaton.feature

```diff
@@ -0,0 +1,54 @@
+@odc-2148
+Feature: Export of application
+              As a user, I have an unmanaged application which I want to export. I'd like to be able to later add that code to git or some shared location so that I can share with others, or import into a new cluster or same cluster but different project, or be able to apply updates to an existing application.
+
+
+
+    #The operator mentioned below is not yet available on operator hub
+        Background:
+            Given user has installed Gitops primer Operator
+              And user has created or selected namespace "aut-export-application"
+
+
+        @smoke @to-do
+        Scenario: Export application button in topology: EA-02-TC01
+            Given user is at Developer perspective
```

Comment:
Move it to BackGround section and remove it from here

Reply:
Updated